net-im/psimedia/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>rion4ik@gmail.com</email>
+		<name>Sergey Ilinykh</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name='extras'>Enables extra non official patches</flag>
```

Comment:
extra extra ;) - rather describe it similar to `net-im/psi` as `Enable unofficial Psi+ patches`?

Please use double quotes everywhere in this file.

Reply:
there are no patches now but rewritten